go/vt/mysqlctl/cephbackupstorage/ceph.go

```diff
@@ -88,7 +88,7 @@ func (bh *CephBackupHandle) AddFile(ctx context.Context, filename string, filesi
 
 		// Give PutObject() the read end of the pipe.
 		object := objName(bh.dir, bh.name, filename)
-		_, err := bh.client.PutObject(bucket, object, reader, "application/octet-stream")
+		_, err := bh.client.PutObject(bucket, object, reader, -1, minio.PutObjectOptions{ContentType: "application/octet-stream"})
```

Comment:
The `PutObjectOptions` look fine to me, but maybe `PutObjectWithContext` should be preferred to pass on the context.

Reply:
thanks for the suggestion. I have added that.